exercism/wordy/wordy.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  WordProblem: class WordProblem {
+    constructor(input){
+      this.input = input,
+      this.operands = {
+        'plus': '+',
+        'minus': '-',
+        'multiplied': '*',
+        'divided': '/'
+      },
+      this.sequence = []
+
+    }
+
+    answer() {
+      let splitString = this.input.replace('?',"")
+      .replace(/by/g,'').split(' ').slice(2)
+      let operandsKeys = Object.keys(this.operands)
+      let match = false
+      operandsKeys.forEach(operand => {
+        if (splitString.includes(operand)) {
+          match = true
+        }
+      })
+      if (!match) {
+        throw new Error()
+      }
+      let mathOperators = splitString.map((x) => {
+        if (this.operands.hasOwnProperty(x)) {
+          return this.operands[x]
+        } else {
+          return x
+        }
+      })
+      let result = mathOperators.join(' ')
```

Comment:
Omit temp variable, and use inline:
```js
return eval( mathOperators.join( ' ' )
```
Not a fan of the use of `eval` as it exposes some risk - users can potentially enter arbitrary JS in the UI, and have it evaluated on server...